bokehjs/src/less/_mixins.less

```diff
@@ -1,5 +1,5 @@
 @font-size: 12px;
-@line-height: 20/14;
+@line-height: 20 ./ 14;
```

Comment:
@philippjfr, if you use division operator in panel's stylesheets, then you will have to replace `/` with `./`, because `/` is now part of CSS specification (compact font and line height spec).